samples/openthread/cli/harness/nrf52840-ncs.py

```diff
@@ -1,3 +1,9 @@
+/*
```

Comment:
I see three issues here:
1. I think #!/usr/bin/env python must be the first line in the file, the copyright may come after
2. The file already has the copyrights done the proper way. This is not a nordic file in fact, so I am not sure this should be changed
3. The copyrights should be in python style comments not C